components/google_drive/actions/add-file-sharing-preference/add-file-sharing-preference.js

```diff
@@ -0,0 +1,96 @@
+const googleDrive = require("../../google_drive.app");
+
+module.exports = {
+  key: "google_drive-add-file-sharing-preference",
+  name: "Add File Sharing Preference",
+  description: "Add File Sharing Preference",
+  version: "0.0.12",
```

Comment:
We need to figure out a better way to handle this, but the registry version should start at `0.0.1` for all of these actions. It's hard when you're constantly running `pd publish` and have to increment the version in your own account. We plan to ship a `pd dev` for actions at some point, which should facilitate testing.